tap_exacttarget/endpoints/events.py

```diff
@@ -87,6 +87,13 @@ def sync_data(self):
                                              'EventDate',
                                              event.get('EventDate'))
 
+                        if event.get('SubscriberKey') is None:
+                            LOGGER.info("SubscriberKey is NULL so ignoring {} record with SendID: {} and EventDate: {}"
+                                    .format(event_name,
+                                            event.get('SendID'),
+                                            event.get('EventDate')))
+                        continue
```

Comment:
should be indented to align with LOGGER.info

Reply:
Fixed

Reply:
hmm, i meant the continue should be aligned...